soda/spark/soda/data_sources/spark_data_source.py

```diff
@@ -59,6 +60,7 @@ def hive_connection_function(
         port=port,
         database=database,
         auth=auth_method,
+        configuration=configuration,
```

Comment:
Can you please use `kwargs` instead? Otherwise it will break other connection functions